hack/lib/vars.bash

```diff
@@ -51,15 +49,8 @@ export UPGRADE_OCP_IMAGE="${UPGRADE_OCP_IMAGE:-}"
 
 export INSTALL_PREVIOUS_VERSION="${INSTALL_PREVIOUS_VERSION:-"false"}"
 
-function latest_channel_from_metadata {
-  local annotations_file channels
-  annotations_file="$(dirname "${BASH_SOURCE[0]}")/../../olm-catalog/serverless-operator/metadata/annotations.yaml"
-  channels="$(yq r "$annotations_file" 'annotations."operators.operatorframework.io.bundle.channels.v1"')"
-  # Return the last channel.
-  echo "$channels" | awk -F"," '{ print $NF }'
-}
 
-export OLM_CHANNEL="${OLM_CHANNEL:-"$(latest_channel_from_metadata)"}"
+export OLM_CHANNEL="${OLM_CHANNEL:-$(metadata.get 'olm.channels.alternatives[]')}"
```

Comment:
I didn't see "alternatives" anywhere else. Should this be "list" ?

Reply:
Yup. It was named like that. It slipped.